### PR TITLE
Add PlayerGamePages and socket.io integration.

### PR DESCRIPTION
**Pull request by prorick**
hi :)

### CODE REVIEW

**Comment by anh65498** [comment]
I'm not sure how to review this PR. Did you put your <AnswerPage> in App.js to build and test it?

File: frontend/src/styles/PlayerAnswerButton.module.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
+.answerButtonBlueStar {
+    position: absolute;
+    width: 500px;
+    height: 300px;
+    left: 745px;
+    top: 125px;
```
Comment (anh65498) [inline frontend/src/styles/PlayerAnswerButton.module.scss]: try to use `rem` as much as possible to create a responsive UI on different screen size. Exception would be border radius, border width, or elements you want to have a fixed size across different screen sizes.

File: frontend/src/styles/PlayerAnswerButton.module.scss [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    background: #01FFF4;
+    border-radius: 17px;
+    color:  #27163F;
+    border-width: 0px;
+}
+
+.answerButtonPinkCircle {
+    position: absolute;
+    width: 500px;
+    height: 300px;
```
Comment (anh65498) [inline frontend/src/styles/PlayerAnswerButton.module.scss]: There's a lot of repetition of these 3 lines of code in your answer buttons. Could you make them into another class so you don't repeat these 3 lines on every answer button? :)

**Comment by anh65498** [comment]
Hi Prothit :) It's a Nice PR with clean code. 👏  Please try to make a more descriptive PR description of what you've done next time.

File: frontend/src/styles/PlayerAnswerButton.module.scss [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+.AnswerGrid {
+    min-height: 100%;
+    min-width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: center;
+
+   
+
```
Comment (anh65498) [inline frontend/src/styles/PlayerAnswerButton.module.scss]: Remove extra space in your files.

**Comment by jnarezo** [comment]
Issues with Button.js from the GameCodePage are preventing compilation. Otherwise, all code is now available for the player to join a room and play the game until the end. Still missing the EndGamePage, however.

The game is accessible at the route `/player/room/:id`.

**Review by anh65498 — APPROVED** [review]

**Pull request merged**